Source/player.cpp

```diff
@@ -1578,15 +1580,15 @@ void __fastcall StartPlrHit(int pnum, int dam, BOOL forcehit)
 
 	drawhpflag = TRUE;
 	if (dam >> 6 >= plr[pnum]._pLevel || forcehit) {
-		int dir = plr[pnum]._pdir;
+		int pd = plr[pnum]._pdir;

```

Comment:
```suggestion
		pd = plr[pnum]._pdir;
```

Fix nested variable.